drivers/pinmux/pinmux_stm32.c

```diff
@@ -383,6 +383,8 @@ int stm32_dt_pinctrl_remap(const struct soc_gpio_pinctrl *pinctrl,
 #endif
 	}
 
+	LL_APB2_GRP1_DisableClock(LL_APB2_GRP1_PERIPH_AFIO);
```

Comment:
For now, AFIO clock is not taken into account in gpio clocks.
So disabling it, here will prevent any further GPIO programmation, such as button.
So I agree current situation is not clean, but I'd prefer we limit changes on that part right now (and postpone to V3.0.0)

Reply:
so, removing the commit